README.md

```diff
@@ -1,27 +1,35 @@
-# datadog-mobile-app-upload
+# Datadog Continuous Testing for Bitrise
 
-placeholder description
+<!-- TODO add link to marketplace after we publish the step -->
+<!-- [![Visual Studio Marketplace Version]()][1001]  -->
+[![Build Status](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4/status.svg?token=480MdFpG78E6kZASg5w1dw&branch=main)](https://app.bitrise.io/app/7846c17b-8a1c-4fc7-aced-5f3b0b2ec6c4)
```

Comment:
I noticed the link to the build status was wrong in the other PR, but correct here 🤔 

https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/pull/12

Reply:
It was correct but actually pointing to the wrong place 😅